e2e-tests/playwright.config.ts

```diff
@@ -9,7 +9,7 @@ import { defineConfig, devices } from '@playwright/test';
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  timeout: 60000,
+  timeout: 120000,
```

Comment:
I think a 60-second timeout is a reasonable limit. If we need more time, we could increase it to 70 or 80 seconds, but 120 seconds for all actions seems too long. It might be better to adjust the timeout on individual actions, depending on the specific action.